src/PREfastXmlSarifConverter/PREfastXmlSarifConverter.cpp

```diff
@@ -402,24 +402,24 @@ HRESULT __stdcall Convert(const std::deque<XmlDefect> defectList, BSTR bstrOutpu
     SarifIssueLog issueLog;
     issueLog.SetVersion(L"0.4");
 
-    // Set ToolInfo
-    SarifToolInfo toolInfo;
-    toolInfo.SetName(L"PREfast");
-    toolInfo.SetFullName(L"PREfast Code Analysis");
-    toolInfo.SetVersion(L"14.0.0");
+    // Set Tool
+    SarifToolInfo tool;
```

Comment:
I'm confused. The OM never had a thing called "SarifToolInfo". What is this, and why does it still have "info" in its name?